src/components/Avatar/Avatar.story.jsx

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { text, number } from '@storybook/addon-knobs';
+import Avatar from './Avatar';
+
+const stories = storiesOf('Avatar', module);
+
```

Comment:
use the `show-in-tests` and `hide-in-tests` sections here as we did with the button. For the `show-in-test` make sure it renders an avatars that have the default size and then step up by 10 until you reach the max. will also want to check the default image url and letter size in tests

Reply:
basically what you did in 👇 snapshot